interpretation/od_analyzer.py

```diff
@@ -0,0 +1,143 @@
+import pandas as pd
+import os
+import re
+from matplotlib import pyplot as plt
+import seaborn as sns
+from interpretation.plotting import roc_plot_grid, standard_curve_plot
+from interpretation.report_reader import slice_df, normalize_od, read_output_batch
+import array_analyzer.extract.constants as constants
+
+def read_config(input_dir):
```

Comment:
Comment on the config file itself: in general plotting settings, "split plots by" properties may not be intuitive to find by the instruction, "pysero spot fitting metadata". Maybe we should point to the example file name: "See pysero spot fitting metadata for available properties (e.g. pysero_output_data_metadata.xlsx)"